sentinel-2/ndwi/script.js

```diff
@@ -1,39 +1,20 @@
-//VERSION=3
-//This script was converted from v1 to v3 using the converter API
+//VERSION 3
+function setup() {
+  return {
+    input: ["B03", "B08"],
```

Comment:
Let's add dataMasks here and return `imgVals.concat(samples.dataMask)` as other `script.js` did